query-node/mappings/src/proposals.ts

```diff
@@ -305,6 +307,12 @@ async function parseProposalDetails(
     const asPayload = unwrap(specificDetails.payload)?.objectCreationParams
     details.payloadHash = asPayload && bytesToString(asPayload.ipfsContentId)
 
+    return details
+  } else if ((proposalDetails as RuntimeProposalDetails_V2002).isSetPalletFozenStatus) {
+    const details = new UpdatePalletFrozenStatusProposalDetails()
+    const specificDetails = (proposalDetails as RuntimeProposalDetails_V2002).asSetPalletFozenStatus
+    details.frozen = specificDetails[0].isTrue
+    details.pallet = specificDetails[1].type
```

Comment:
I realized I made a mistake in original PR, although this kind of works because there is a single enum variant.
The correct way would be (assuming the variants have no encapsulated data) would be:

```suggestion
    details.pallet = specificDetails[1].toString()
```

Reply:
@mnaamani yeah I also thought it was a bit strange but I assumed that it was ok to commit them formatted, should I revert that?